scala/scala.bzl

```diff
@@ -162,9 +162,25 @@ def _compile(ctx, cjars, dep_srcjars, buildijar):
     all_srcjars = set(srcjars + list(dep_srcjars))
     # look for any plugins:
     plugins = _collect_plugin_paths(ctx.attr.plugins)
+
+    if (hasattr(ctx.attr, 'enable_dependency_analyzer')
+        and ctx.attr.enable_dependency_analyzer):
+      enable_dependency_analyzer = ctx.attr.enable_dependency_analyzer
```

Comment:
isn't this always true here? Can we just write `enable_dependency_analyzer = True` here?